app/src/main/java/com/belatrixsf/connect/adapters/EventListAdapter.java

```diff
@@ -77,11 +77,18 @@ public void onBindDataViewHolder(RecyclerView.ViewHolder holder, int position) {
             eventViewHolder.eventTitleTextView.setText(title);
             eventViewHolder.eventDateTextView.setText(formattedDate);
             eventViewHolder.itemView.setTag(event);
-            ImageFactory.getLoader().loadFromUrl(event.getPicture(),
+            ImageFactory.getLoader().loadFromUrl(
+                    event.getPicture(),
                     eventViewHolder.pictureImageView,
                     ImageLoader.ImageTransformation.BORDERED_CIRCLE,
-                    eventViewHolder.pictureImageView.getResources().getDrawable(R.drawable.event_placeholder)
+                    null,
+                    eventViewHolder.pictureImageView.getResources().getDrawable(R.drawable.contact_placeholder)
             );
+            /*ImageFactory.getLoader().loadFromUrl(event.getPicture(),
```

Comment:
i forgot to comment that there is a change with the image transformation in the event list and event detail views.